test/cypress/integration/settings/switch-psp.cy.test.js

```diff
@@ -250,22 +320,22 @@ describe('Switch PSP settings page', () => {
       it('should create a charge and continue to charges next url on success', () => {
         cy.task('setupStubs', [
           ...getUserAndAccountStubs('smartpay', true, [
-            {
-              payment_provider: 'smartpay',
-              state: 'ACTIVE',
-              id: currentCredentialId,
-              external_id: currentCredentialExternalId
-            },
-            {
-              payment_provider: 'worldpay',
-              state: 'ENTERED',
-              id: switchingToCredentialId,
-              external_id: switchingToCredentialExternalId
-            }
-          ],
-          null,
-          true,
-          2),
+              {
```

Comment:
This re-formatting is being complained by IDE. Similar limiting issues across.

<img width="806" alt="image" src="https://user-images.githubusercontent.com/40598480/211573786-d2aa16c2-f0b5-4b36-a7b5-32a9ef93ae12.png">

Reply:
Issues with my Standard linting in Intelli J - this is fixed now.